packages/mermaid/src/diagrams/flowchart/parser/flow.jison

```diff
@@ -540,16 +540,22 @@ styleStatement:STYLE SPACE idString SPACE stylesOpt
 linkStyleStatement
     : LINKSTYLE SPACE DEFAULT SPACE stylesOpt
           {$$ = $LINKSTYLE;yy.updateLink([$DEFAULT],$stylesOpt);}
+    | LINKSTYLE SPACE MINUS SPACE stylesOpt
+          {$$ = $LINKSTYLE;yy.updateLink([yy.getEdges().length-1],$stylesOpt);}
```

Comment:
would not be better to wrap
```
edges.length - 1
```
in a function, e.g. `lastEdgeIndex` and put it in DB, instead of making some calculations here?

Multiple occurrences of the same code